src/test/java/net/snowflake/client/jdbc/structuredtypes/StructuredTypesArrowJsonCompatibilityLatestIT.java

```diff
@@ -54,14 +51,29 @@ public static void closeConnections() throws SQLException {
   }
 
   @Test
-  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
   public void testRunAsGetString() throws SQLException {
     withFirstRow(
         connections.get(queryResultFormat),
         selectSql,
         (resultSet) -> assertGetStringIsCompatible(resultSet, expectedStructureTypeRepresentation));
   }
 
+  @Test
+  public void testRunAsGetObject() throws SQLException {
```

Comment:
we are fetching and getting the same data for Object and Bytes - can we run bot (or even more tests) on the same result set?